data/indigenous.txt

```diff
@@ -8,6 +8,49 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+ship "Subsidurial"
+	noun "creature"
+	sprite "ship/subsidurial"
+	thumbnail "thumbnail/subsidurial"
+	"never disabled"
+	attributes
+		category "Light Freighter"
+		"hull" 11600
+		"required crew" 3
+		"bunks" 3
```

Comment:
These need zero crew, otherwise killing one will eternally piss off the Archons, which could happen before the player even gets anywhere near meeting an Archon.

Reply:
Maybe the Archons are secretly watching over all natively space-faring life?

Reply:
I think this now counts as resolved, as this is now a person that players will occasionally see in uninhabited systems rather than a "frequently" occurring ship. (also, it has been dropped to crew 1 / bunks 1, so it'll have a lower impact on ratings when killed, too.)